### DISCUSSION TITLE
Where to send LibreTranslate fix suggestion to?

### DISCUSSION THREAD

**Discussion by rffontenelle** [WeblateOrg/weblate#10483] [Q&A] [answered]
I found a terrible automatic suggestion by LibreTranslate when using Hosted Weblate and I would like to suggest a fix. Where and how I can provide this contribution?

**Comment by nijel** [comment, marked as answer]
See https://community.libretranslate.com/t/how-to-help-libretranslate-as-a-user-not-developer/775

**Reply by rffontenelle** [reply]
Quoting:
> While models are pre-trained, you can submit suggestions on [https://libretranslate.com](https://libretranslate.com/) by pressing the “Edit” button (pencil icon) if a translation is wrong. The suggestions are not immediately taken in consideration, but they might be included in a future language model update.
